docs/semgrep-code/getting-started.md

```diff
@@ -29,333 +28,146 @@ import PlatformSigninGithub from "/src/components/procedure/_platform-signin-git
-* Notifying security teams of findings (results) as well as communicating with other developers by leaving pull or merge request comments in GitHub, GitLab, or BitBucket.
-
-:::tip Suggested workflows 
-* You can use Semgrep Cloud Platform to scan remote repositories (GitHub, GitLab, or BitBucket) and consolidate the findings.
-* You can also use Semgrep Cloud Platform to consolidate findings from a Semgrep CLI scan performed on a **local** machine.
-* Semgrep Code can be integrated into your custom infrastructure without SCP. See [API](/semgrep-cloud-platform/semgrep-api) for details.
-:::
-
-### Scanning a repository
-
-#### Signing in to Semgrep Cloud Platform
-
-<PlatformSigninIntro />
+# Scan your codebase
 
-<Tabs
-    defaultValue="signin-github"
-    values={[
-    {label: 'Sign in with GitHub', value: 'signin-github'},
-    {label: 'Sign in with GitLab', value: 'signin-gitlab'},
-    ]}
->
+Semgrep Code is a static application security testing (SAST) tool that scans your repository for security vulnerabilities. You can scan your repository by integrating Code into your CI/CD pipeline, or you can scan repositories available locally on your machine.
 
-<TabItem value='signin-github'>
+## Use the Semgrep Cloud Platform (SCP) to automate the continuous scanning of your code
 
-<PlatformSigninGithub />
-
-See [Permissions in GitHub](/semgrep-cloud-platform/getting-started/#requested-permissions-for-github-and-gitlab) to learn more about how Semgrep features use requested permissions in GitHub.
-
-</TabItem>
-
-<TabItem value='signin-gitlab'>
-
-<PlatformSigninGitlab />
-
-See [Permissions in GitLab](/semgrep-cloud-platform/getting-started/#requested-permissions-for-github-and-gitlab) to learn more about how Semgrep features use requested permissions in GitLab.
-
-</TabItem>
-
-</Tabs>
+<PlatformAddRepo />
 
-:::tip
-To use Semgrep with your team, [create an organization account](/semgrep-cloud-platform/user-management/). An organization account enables users to share rules and perform triage or remediation as a team.
+:::tip Scan a demo application
+You can start using Semgrep by scanning a demo project that requires only minor
+configuration. See [Learning Semgrep Cloud Platform with a demo project](/semgrep-code/demo-project/)
+for details.
 :::
 
-## Performing a scan
-
-Scanning is Semgrep's primary operation. When you first sign into Semgrep Cloud Platform, it uses a default SAST ruleset selected to enforce best practices for a repository's framework and programming language. You can customize future scans to address your organization's specific practices.
-
-Semgrep Cloud Platform enables users to choose what findings prevent a pull or merge request (PR or MR) from merging into the repository. Setting these blocking and non-blocking rules is achieved through the [Policies page](/semgrep-code/policies).
```

Comment:
We probably want to include this in a future onboarding doc. I'm noting this down so that we can collect material already written for an onboarding doc - I'll resolve this comment independently when I've started prepping the PR for it.